lib/ObjWriter/.nuget/Linux/Microsoft.Dotnet.ObjectWriter.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Microsoft.DotNet.ObjectWriter</id>
+    <version>1.0.0-prerelease</version>
+    <title>Microsoft .NET ObjectWriter</title>
```

Comment:
I don't get it. Do you suggest "Microsoft.ObjectWriter" instead of "Microsoft.DotNet.ObjectWriter"?